programmers/20210506/두-개-뽑아서-더하기.js

```diff
@@ -0,0 +1,18 @@
+function solution(numbers) {
+    let answer = [];
+    for (let i = 0; i < numbers.length; i++) {
+        for (let j = 1; j < numbers.length && i !== j; j++) {
+            let sumNum = numbers[i] + numbers[j];
```

Comment:
하나의 흐름에서 변하지 않는 값은 const로 선언해줍니다
```suggestion
            const sumNum = numbers[i] + numbers[j];
```